apps/src/applab/appStorage.js

```diff
@@ -372,7 +372,7 @@ var handlePopulateKeyValue = function (onSuccess, onError) {
     return;
   }
 
-  if (this.status != 200) {
+  if (this.status !== 200) {
```

Comment:
Similar to comment regarding ajaxSubmit.js, if status can actually be a string or a number, this would get us into trouble.